<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>709</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1344</width>
    <height>709</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1344</width>
    <height>709</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="header" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1351</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QLabel" name="user_ip">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>91</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="user">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>4</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="MainWindow.qrc">:/MainWindow/Resources/user.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="team">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>30</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SimSun-ExtB</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(104, 33, 122);
border:0;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>团队</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submit">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>30</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>SimSun-ExtB</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(104, 33, 122);
border:0;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1351</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPlainTextEdit" name="code">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>1141</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="update">
      <property name="geometry">
       <rect>
        <x>1091</x>
        <y>10</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>上传代码</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="title">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>1141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="plainText">
       <string>title</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QListWidget" name="person_lw">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>211</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:0;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>stackedWidget</zorder>
   <zorder>header</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1344</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
